app/controllers/api/v1/offers_controller.rb

```diff
@@ -146,6 +146,12 @@ def reopen_offer
         render json: @offer, serializer: offer_serializer
       end
 
+      api :PUT, '/v1/offers/1/resume_receiving', "Change Offer to Receiving state"
+      def resume_receiving
+        @offer.update_attributes({ state_event: 'start_receiving' })
```

Comment:
you can probably do `@offer.start_receiving`

Also, on which initial states can you apply this transition ? I assume you can't do that from any states.
If that's the case, make sure there are tests to prove you can't do this for any offer in any state